src/cosmjs.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.CosmJSOfflineSigner = exports.CosmJSOfflineSignerOnlyAmino = void 0;
+class CosmJSOfflineSignerOnlyAmino {
```

Comment:
I think we can merge these two classes into `TrustCosmosWeb3Provider`, one OfflineSigner for all, default `sign` is `signAmino`

Reply:
https://github.com/cosmos/cosmjs/blob/main/packages/proto-signing/src/signer.ts, just two methods